lib/src/Uncertainty/Model/DistributionImplementation.cxx

```diff
@@ -1147,7 +1147,7 @@ Scalar DistributionImplementation::computeProbabilityContinuous(const Interval &
 Scalar DistributionImplementation::computeProbabilityContinuous1D(const Scalar aa, const Scalar bb) const
 {
   // Use adaptive multidimensional integration of the PDF on the reduced interval
-  const PDFWrapper pdfWrapper(this);
```

Comment:
does this mean that we can make PDFWrapper wrapper if you removed all the occurences outside DistributionImplementation ?